code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1996,6 +1996,25 @@
 	center = TRUE
 	dimension_y = 32
 
+/datum/sprite_accessory/wings/clockwork
```

Comment:
move to modular folder

Reply:
i don't know what you mean by modular folder, so i moved it back to disease file until future notice.